app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetController.kt

```diff
@@ -60,7 +59,6 @@ class BloodPressureEntrySheetController @Inject constructor(
         .replay()
 
     return Observable.mergeArray(
-        showBpEntry(replayedEvents),
```

Comment:
It's kind of funny that the commit message says "Move showing of data validation errors when back is clicked...", and the method that gets removed is `showBpEntry`. 😅

Reply:
😂 This is because validating the BP happens in multiple scenarios. One of them was when we click the show BP entry button. This was the last code path that I removed which was triggering the validation logic.